### ISSUE TITLE
Contra 4 crashes melonDS

### ISSUE THREAD

**Issue by BiZkViT96** [Arisotura/melonDS#1281]
The latest versions of the emulator crash when the game loads. This happens regardless of the render - Software and OpenGL. Version 0.9.3 runs the game fine.

**Comment by iamthelaw2041** [comment]
Game works fine. Tested with JIT on and off, and with External and built in BIOS. 

![Capture](https://user-images.githubusercontent.com/94656803/144695952-ef77ba95-8759-4de0-8529-2054e47540ad.PNG)

**Comment by BiZkViT96** [comment]
As I found out, any game starting with PR #[1251](https://github.com/Arisotura/melonDS/actions/runs/1395579960) crashes the emulator. The latest version that works correctly PR#[1238](https://github.com/Arisotura/melonDS/actions/runs/1326623017) I looked [here](https://github.com/Arisotura/melonDS/actions/workflows/build-windows.yml?page=2&query=workflow%3A%22CMake+Build+%28Windows+x86-64%29%22+event%3Apush).

**Comment by RSDuck** [comment]
this is a known issue, it has nothing to do with that specific commit and more with that this commit was the first to be built with SDL 2.0.16.

**Comment by RSDuck** [comment]
this should be fixed with latest SDL, can you confirm?

**Comment by BiZkViT96** [comment]
Yes, apparently the problem has been solved. A microphone icon appears on the panel indicating that the emulator uses it and the games load and work normally.
